src/uct/cuda/cuda_copy/cuda_copy_iface.h

```diff
@@ -20,8 +21,8 @@ ucs_status_t _status = UCS_OK;                          \
 do {                                                    \
     cudaError_t _result = (func);                       \
     if (cudaSuccess != _result) {                       \
-        ucs_error("[%s:%d] cuda failed with %d \n",     \
-                  __FILE__, __LINE__,_result);          \
+        ucs_error("%s failed with %d \n",               \
```

Comment:
need to rename macro from CUDA_FUNC to UCT_CUDA_CALL, and use '_func' instead of 'func'

Reply:
fixed